examples/oidc/README.md

```diff
@@ -0,0 +1,119 @@
+# Secure a service with OIDC authentication
```

Comment:
We should maybe also point out more clearly, that this is explicitly tailored for the use case when kubernetes is *not* configured to use OIDC for authentication.

If kubernetes *is* configured with OIDC, one should still use the existing delegating authn provider (which passes the OIDC ID token to the token review).